src/app/index.ts

```diff
@@ -1,15 +1,17 @@
+import h from "hyperscript"
 import { Explorer } from "~explorer"
 import { NavigationBar } from "~navigation-bar"
 
 import { rootFolder, setCurrentFolder } from "./state"
-
 import "./app.css"
 
-export const appElement = document.getElementById("app") as HTMLElement
+export let appElement: HTMLElement | null = null
```

Comment:
Although I am exporting it here, it is probably better to import it directly as rarely as possible.